### ISSUE TITLE
Methods with primitives should include this.primitiveFailed or other fallback code

### ISSUE THREAD

**Issue by muellmusik** [supercollider/supercollider#2879]
Some methods which call primitives have nothing after the primitive invocation, which means that any errors or exceptions will be silenced. They should include either ```^this.primitiveFailed``` or in special cases sclang fallback code. e.g.

```*scVersionMajor   { _SC_VersionMajor }```

@jamshark70 has offered this as a list of candidates to be checked (but probably good to check all primitives in any case).

```
(
m = List.new;
var returnSelf = Int8Array[0xF4];
Class.allClasses.do { |cl|
	cl.methods.do { |method|
		if(method.primitiveName.notNil and: { method.code == returnSelf }) {
			m.add(method);
		}
	}
};
m.do(_.postln); ""
)

ArrayedCollection:indexedSize
ArrayedCollection:size
ArrayedCollection:maxSize
Boolean:keywordWarnings
Boolean:trace
Char:ascii
Char:toUpper
Char:toLower
Char:isAlpha
Char:isAlphaNum
Char:isPrint
Char:isPunct
Char:isControl
Char:isSpace
Char:isDecDigit
Char:isUpper
Char:isLower
Class:dumpClassSubtree
Float:as32Bits
Float:high32Bits
Float:low32Bits
FunctionDef:dumpByteCodes
FunctionDef:numArgs
FunctionDef:numVars
FunctionDef:varArgs
FunctionDef:dumpContexts
Integer:nextPowerOfTwo
Integer:isPowerOfTwo
Integer:leadingZeroes
Integer:trailingZeroes
Integer:numBits
Integer:log2Ceil
Integer:grayCode
Integer:nthPrime
Integer:prevPrime
Integer:nextPrime
Integer:indexOfPrime
Integer:isPrime
Integer:exit
Meta_AppClock:prSchedNotify
Meta_Class:allClasses
Meta_Date:seed
Meta_Font:availableFonts
Meta_LanguageConfig:currentPath
Meta_LanguageConfig:includePaths
Meta_LanguageConfig:excludePaths
Meta_LanguageConfig:postInlineWarnings
Meta_Main:scVersionMajor
Meta_Main:scVersionMinor
Meta_Main:scVersionPostfix
Meta_Pen:matrix
Meta_Process:tailCallOptimize
Meta_Process:elapsedTime
Meta_Process:monotonicClockTime
Meta_QtGUI:cursorPosition
Meta_ScIDE:connected
Meta_Thread:primitiveError
Meta_Thread:primitiveErrorString
Object:dump
Object:totalFree
Object:largestFreeBlock
Object:gcDumpGrey
Object:gcDumpSet
Object:gcInfo
Object:gcSanity
Object:canCallOS
Object:prHalt
Object:dumpBackTrace
Object:getBackTrace
Object:crash
Object:stackDepth
Object:dumpStack
Object:dumpDetailedBackTrace
Object:pr_asCompileString
Platform:recompile
Platform:userHomeDir
Platform:systemAppSupportDir
Platform:userAppSupportDir
Platform:systemExtensionDir
Platform:userExtensionDir
Platform:userConfigDir
Platform:resourceDir
Platform:ideName
String:hash
String:asCompileString
String:prPostln
String:prPost
Symbol:asClass
Symbol:isMap
Thread:randData
Thread:failedPrimitiveName
```

**Comment by patrickdupuis** [comment]
I can take care of this in a weeks time.

**Comment by patrickdupuis** [comment]
fixed in #2908
